mizar/get_interface.py

```diff
@@ -0,0 +1,11 @@
+import os
+import logging
+logger = logging.getLogger()
+
+def get_itf():
+    default_itf = os.popen("route | grep '^default' | grep -o '[^ ]*$'").read().split('\n')[0]
+    if "NET_ITF" in os.environ:
```

Comment:
Suggest changing the environment variable with prefix "Mizar" to: 1) make it clear that it's for Mizar. 2) reduce the change of name conflicts.  Such as "MIZAR_NET_INTERFACE".

Reply:
Updated the environment variable name as per the suggestion.